### DISCUSSION TITLE
What's you dev env like on Windows? Is Laragon the best choice over XAMPP?

### DISCUSSION THREAD

**Discussion by iannngit** [statamic/cms#7147] [General]
Hi all,

A general question about how you make your Statamic 3 websites running on local. I've been using Laragon and it's good. But I'm wondering would there be better solutions please.

**Comment by jasonvarga** [comment]
Sounds like you must be using Windows, right?

On Mac you can't beat Laravel Valet. There is a Windows version, but didn't even know it existed until just now when I Googled for it. 😄 If it works so well on Mac, maybe you'll be happy with the Windows equivalent. https://github.com/cretueusebiu/valet-windows (Again, I haven't tried it so I can't say how well it works.)

**Comment by twd3** [comment]
Hands down it's the Windows version of Valet! I'm 100% Windows and have been using this dev env for years. Just install PHP, Laravel Valet and you're good!
